.github/workflows/goreleaser.yml

```diff
@@ -62,22 +61,21 @@ jobs:
           key: ${{ runner.os }}-osxcross-${{ env.OSX_CROSS_MACOS_SDK_VERSION }}
           path: |
             ${{ env.OSXCROSS_DIR }}/target/bin
-
       - name: Build OSX Cross Compiler Tool Chain
         if: steps.osxcross-cache.outputs.cache-hit != 'true'
         run: |
           cd $OSXCROSS_DIR
           SDK_VERSION=$OSX_CROSS_MACOS_SDK_VERSION UNATTENDED=yes ./build.sh
-
       - name: Add OSX Cross Compiler Tool Chain to Path
         run: |
           echo "$OSXCROSS_DIR/target/bin" >> $GITHUB_PATH
 
       - name: Run GoReleaser
         id: run-goreleaser
-        uses: goreleaser/goreleaser-action@8f67e590f2d095516493f017008adc464e63adb1 # v4.1.0
+        uses: goreleaser/goreleaser-action@286f3b13b1b49da4ac219696163fb8c1c93e1200 # v4.1.0
```

Comment:
Can we please update the # 4.1.0 comment to the version that the SHA should resolve to?